### ISSUE TITLE
add a "See details" button in campaign widget

### ISSUE THREAD

**Issue by altatof** [Charitable/Charitable#35]
the nicest way would be an option to see a "details" button and/or the "donate" button

**Comment by ericnicolaas** [comment]
I think this makes sense. I see it working as an additional parameter to pass to the `campaigns` shortcode. Perhaps something like this: 
```
// Display "Details" button
[campaigns button=details] 

// Display "Donate" button
[campaigns button=donate]

// No button
[campaigns button=false]
```

@altatof: What do you think if that implementation? 

Cheers, 
Eric

**Comment by altatof** [comment]
the implementation sounds good to me, dont forget the default value (to Donate).
(for the little story, it's my partner in life and in work who told me about the poor details of my campaign... in fact she did not guess that she has to click on the widget to see details, she just clicked on the Donate big button and was surprised to see just the donation form without campaign details

**Comment by ericnicolaas** [comment]
Fixed in fd68a3a38159c9792f2f5bef289d897f603ac197
